src/common/perf_counters.h

```diff
@@ -233,6 +233,11 @@ class PerfCounters
 
   ~PerfCounters();
 
+  std::atomic<bool> time_filtered = { false };
```

Comment:
@cbodley Should making each of these 3 new PerfCounters variables atomic address your concern of these variables being thread safe? They're being read and written to in `rgw_perf_counters.cc` in the op_counters functions that don't have locks.

Reply:
i think it's important that we don't make `PerfCounters` more expensive for daemons like ceph-osd which aren't using a cache, so they shouldn't have to pay for extra atomics

i see that `time_filtered` and `time_alive` both remain constant after initialization, so we shouldn't need those to be atomic. as long as `time_filtered` is always false for ceph-osd, they won't need to read the atomic `last_updated`

Reply:
Ack, I'll make that change. I do know that we talked about making these op counters configurable at runtime but it's best to not think about that yet.